eRecruiter.Api/Responses/Applicant/ApplicantHistoryResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿using eRecruiter.Api.Parameters;
+
+namespace eRecruiter.Api.Responses
+{
+    public class ApplicantHistoryResponse : ApplicantHistoryParameter
```

Comment:
Since we are never returning status code 201 and the requests just accept classes as return objects I would leave it that way.